### ISSUE TITLE
Consolidate all user facing changes in 6.6 to aid release, blog and testing

### ISSUE THREAD

**Issue by OliverHannan** [digital-preservation/droid#852]
To expedite 6.6.0 RC2 can we have a list of all user facing changes?

The reason for this is it, will aid the efficient internal QA testing but also a blog post (to alert DROID community generally) and future user research.

[They are neatly summed up here](https://github.com/digital-preservation/droid/pull/631) but it'd be good to double check. We can then move to describe, test and screenshot for the reasons given.  

This ticket will be done when we have a consolidated list.

**Comment by steve-daly** [comment]
Yes, https://github.com/digital-preservation/droid/pull/631 has some excellent write up of the command-line changes.  It would be useful to look for things like this too https://github.com/digital-preservation/droid/pull/631/commits/16fa95561875af91cc5479d0e8019ecceae7d56f as that's a useful change to 'profile mode' that we'd want to re-test and communicate.

**Comment by OliverHannan** [comment]
Hi @paulwellsagilej, as part of this tickets getting over the line and getting our RC out can you sense check the testing spreadsheet I shared? This isn't so much the results, more that the testing is sufficient to cover what we now know about the large outstanding PR.
